README.md

```diff
@@ -1,37 +1,148 @@
-# Ruby Class Variables And Class Methods
+# Class Variables and Class Methods
 
-## Outline
+## Objectives
 
-All objects are a combination of data and logic encapsulated by a singular object. We're used to seeing this on an instance level, on instances of classes.
+1. Define a class variable. 
+2. Define a class method. 
+3. Understand why and when to use class variables and methods. 
+4. Understand the scope of class variables and class methods. 
 
+## Introduction: Classes are Objects Too
+
+All objects are bundles of data and logic––or attributes and behavior. We understand this to be true of instances of a class. Each instances contains attributes or properties as well as methods that can enact behaviors. 
+
+For example, let's say we have a class, `Album`. Every individual album instance should have a property, `@release_date` and an instance method `#release_date` that exposes or reveals that property. 
```

Comment:
use the term "attribute" instead of property. Also don't say an attribute `@release_date`. That's the backing instance variable. It should be an attirbute `release_date`.

Nitpicking, but this stuff is super confusing when you start